client/src/components/FeedbackObject/index.js

```diff
@@ -0,0 +1,165 @@
+	const handleDelete = (feedbackId) => {
+		fetch(`/api/students/${student._id}/${feedbackId}`,
+			{
+				method: "DELETE",
+				headers: { "Content-type": "application/json" },
+			})
+			.then((res) => res.json())
+			.catch((error) => console.log(error));
+	};
+
+	// Populate new object with edited data editable
+	const handleEdit = (e) => {
+		setCurrentFeedback({ ...currentFeedback, [e.target.name]: e.target.value });
+	};
+
+	return (
+		<div className="previous-feedback-list">
+			{Object.keys(currentFeedback).length && Object.keys(currentFeedback)
+				.map((property, index) => {
+					if (property === "module") {
+						return (
+							<div className="previous-feedback-module" key={index}>
+								<p className="feedback-input-head">Feedback module:</p>
+								{!isEditing
+									? <p>{currentFeedback[property]}</p>
+									: <select
+										name={property}
+										value={currentFeedback[property]}
+										onChange={handleEdit}>
+										{modules.map((module, index) =>
+											<option key={index} value={module.name}>{module.name}</option>
+										)}
+									</select>}
+							</div>
+						);
+					}
+					if (property === "title") {
+						return (
+							<div className="previous-feedback-title" key={index}>
+								<p className="feedback-input-head">Feedback title:</p>
+								<input
+									className="feedback-title-input"
+									name={property}
+									value={currentFeedback[property]}
+									onChange={handleEdit}
+									disabled={!isEditing}
+								/>
+							</div>
+						);
+					}
+					if (property === "text") {
+						return (
+							<div className="previous-feedback-text" key={index}>
+								<p className="feedback-input-head">Feedback:</p>
+								<textarea
+									className="previous-feedback-text-input"
+									name={property}
+									value={currentFeedback[property]}
+									onChange={handleEdit}
+									disabled={!isEditing}
+								/>
+							</div>
+						);
+					}
+					if (property === "time") {
+						return (
+							<div className="previous-feedback-time" key={index}>
+								<input
+									className="previous-feedback-time-input"
+									name={property}
+									value={timeDifference(Date.now(), currentFeedback[property])}
+									onChange={handleEdit}
+									disabled
+								/>
+							</div>
+						);
+					}
+					if (property === "mentor") {
+						return (
+							<div className="previous-feedback-mentor" key={index}>
+								<p className="feedback-input-head">Given by:</p>
+								<input
+									className="previous-feedback-mentor-input"
+									name={property}
+									value={currentFeedback[property]}
+									onChange={handleEdit}
+									disabled
+								/>
+							</div>
+						);
+					}
+				})}
+			<div className="edit-delete-buttons">
+				<Button
+					content={isEditing ? "Save" : "Edit"}
+					handleClick={() => {
+						setIsEditing(!isEditing);
+						isEditing && saveFeedback(currentFeedback.id);
```

Comment:
Content equation is wrong. if isEditing TRUE the button should be SAVE. and on the first render it comes as SAVE and editable.

Reply:
You are absolutely right. 
It is "Save" when isEditing is True. 
And the first render it is False line no.11: 
const [isEditing, setIsEditing] = useState(false);